README.md

```diff
@@ -39,10 +39,9 @@ Prerequisite: [Node 20+](https://www.npmjs.com/package/node/v/20.11.1)
 
 ### Stake Partial ETH :diamond_shape_with_a_dot_inside:
 
-This code sample creates an ETH staking workflow. View the full code sample [here](examples/ethereum/create-workflow.ts)
+This code sample helps stake partial ETH (non-multiples of 32 ETH). View the full source [here](examples/ethereum/create-workflow.ts)
```

Comment:
```suggestion
This code sample helps stake ETH via partial staking. View the full source [here](examples/ethereum/create-workflow.ts)
```

Reply:
Ooops - merged and later saw this. Can incorporate in next PRs.